app/controllers/api/databases.php

```diff
@@ -2790,3 +2873,98 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
+        $queries = Query::parseQueries($queries);
+        $queries[] = Query::equal('databaseId', [$databaseId]);
+
+        $cursor = Query::getByType($queries, Query::TYPE_CURSORAFTER, Query::TYPE_CURSORBEFORE);
+        $cursor = reset($cursor);
+        if ($cursor) {
+            $documentId = $cursor->getValue();
+            $cursorDocument = $dbForProject->getDocument('slowQueries', $documentId);
+            if ($cursorDocument->isEmpty()) {
+                throw new Exception(Exception::GENERAL_CURSOR_NOT_FOUND, "Slow query '{$documentId}' for the 'cursor' value not found.");
+            }
+            $cursor->setValue($cursorDocument);
+        }
+
+        $response->dynamic(new Document([
+            'documents' => $dbForProject->find('slowQueries', $queries),
+            'total' => $dbForProject->count('slowQueries', Query::groupByType($queries)['filters'], APP_LIMIT_COUNT),
+        ]), Response::MODEL_DOCUMENT_LIST);
+    });
+
+App::get('/v1/databases/:databaseId/slow-queries/:documentId')
+    ->desc('Get Slow Query Document')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('scope', 'documents.read')
+    ->label('usage.metric', 'documents.{scope}.requests.read')
+   // ->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.method', 'getSlowQuery')
+    ->label('sdk.description', '/docs/references/databases/get-slow-query.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_DOCUMENT)
+    ->param('databaseId', '', new UID(), 'Database ID.')
+    ->param('documentId', '', new UID(), 'Document ID.')
+    ->inject('response')
+    ->inject('dbForProject')
+    ->action(function (string $databaseId, string $documentId, Response $response, Database $dbForProject) {
+        $document = $dbForProject->getDocument('slowQueries', $documentId);
+        if ($document->isEmpty() || $document->getAttribute('databaseId') !== $databaseId) {
+            throw new Exception(Exception::DOCUMENT_NOT_FOUND);
+        }
+
+        $response->dynamic($document, Response::MODEL_DOCUMENT);
+    });
+
+App::delete('/v1/databases/:databaseId/slow-queries/:documentId')
+    ->desc('Delete Slow query Document')
+    ->desc('List  Documents')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('scope', 'documents.write')
+    ->label('usage.metric', 'documents.{scope}.requests.delete')
+    //->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.method', 'deleteSlowQuery')
+    ->label('sdk.description', '/docs/references/databases/delete-slow-query-document.md')
```

Comment:
Let's rename this file to `delete-slow-query` to be consistent with the others

Reply:
done!